server/src/test/java/org/spine3/server/projection/ProjectionRepositoryShould.java

```diff
@@ -600,5 +659,25 @@ private ManualCatchupProjectionRepository(BoundedContext boundedContext,
         public void apply(ProjectCreated event, EventContext eventContext) {
             // NOP
         }
+
+        /**
+         * {@inheritDoc}
+         *
+         * <p>Overrides to expose the method into this test class.
```

Comment:
`to expose smth. to` (or `to expose smth.` which is not the case) — see [this article](http://www.dictionary.com/browse/expose). Please fix.